### PR TITLE
Quast: init at 5.0.2

### PR DESCRIPTION
**Pull request by bzizou**
###### Motivation for this change
This tools is used into my laboratory

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/applications/science/biology/quast/default.nix [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+   postFixup = ''
+   for file in $(find $out -type f -type f -perm /0111); do
+       old_rpath=$(patchelf --print-rpath $file) && \
+       patchelf --set-rpath $old_rpath:${stdenv.cc.cc.lib}/lib $file || true
+   done
+  '';
+
+  dontPatchELF = true;
+
+  doCheck = false;
```
Comment (risicle) [inline pkgs/applications/science/biology/quast/default.nix]: Please leave a comment of explanation.

**Comment by risicle** [comment]
Builds on macos 10.13 & executable appears to work, but no tests so I've no confidence it _actually_ works.

Are you sure you wouldn't prefer to expose the executables as `quast` rather than `quast.py`?

**Comment by bzizou** [comment]
The build of this python application is a bit out of standards and it's been a bit hard to package. Tests are disabled because they need a set of data to be downloaded first, so we must run it by hand. Running tests by hand works, despite some "non-fatal" errors 
I'm going to check if we can easyly change from quast.py to quast...

**Comment by Lassulus** [comment]
ping?

**Comment by bzizou** [comment]
Sorry for the delay... I just added a link `quast` to the main executable `quast.py`. I did not rename it, because there's too many references to <script>.py into the Quast documentation and scripts.

**Review by Lassulus — APPROVED** [review]
tested with nix-review, lgtm

**Pull request merged**